wgsl/index.bs

```diff
@@ -2028,7 +2028,7 @@ Some consequences of the restrictions structure member and array element types a
 <div class='syntax' noexport='true'>
   <dfn for=syntax>struct_body_decl</dfn> :
 
-    | [=syntax/brace_left=] ( [=syntax/struct_member=] [=syntax/comma=] ) * [=syntax/struct_member=] [=syntax/comma=] ? [=syntax/brace_right=]
+    | [=syntax/brace_left=] ( [=syntax/struct_member=] ( [=syntax/comma=] [=syntax/struct_member=] ) * [=syntax/comma=] ? ) ? [=syntax/brace_right=]
```

Comment:
Any reason not to pull the first member out of the conditional grouping?

```suggestion
    | [=syntax/brace_left=] [=syntax/struct_member=] ( ( [=syntax/comma=] [=syntax/struct_member=] ) * [=syntax/comma=] ? ) ? [=syntax/brace_right=]
```

Reply:
You're right, WGSL structs have at least one member.
That should lead to a further simplification:

       brace_left struct_member (comma struct_member)* comma? brace_right

I'll try that.

Reply:
Removed that outer ( ... ) ? construction